wp-shopify.php

```diff
@@ -26,4 +26,132 @@ function pp() {
         }
     }
 
+    // Set rewrite slug on first activation
+    function set_wps_rewrite_slug(){
+        if( get_option('wshop_rewrite_slug') == '' ){
+            update_option( 'wshop_rewrite_slug', 'store' );
+            flush_rewrite_rules();
+        }
+    }
+
+    register_activation_hook( __FILE__, 'set_wps_rewrite_slug' );
+
+    // Define AJAX functions
+    function wps_process_product(){
+
+        // Get the ID of the current Product
+        $id = $_REQUEST['product_id'];
```

Comment:
use $_POST instead of request. We don't want this to be able to happen with a GET request